### ISSUE TITLE
Why validation loss<training loss

### ISSUE THREAD

**Issue by guoyaol** [alfonmedela/TripletSemiHardLoss-PyTorch#2]
excuse me, during the training, why the validation loss is less than training loss

**Comment by HariWu1995** [comment]
because of overfitting? I think you should ask a more detailed question on StackOverflow.

**Comment by manxiaoyang** [comment]
if loss always equals 0     the acc result is very bad

**Comment by alfonmedela** [comment]
If you refer to the accuracy displayed during training, then forget about that because it has nothing to do with the performance of the model. 

Is this the problem or you mean that after training a model that achieves 0 loss, the performance as a classifier is very poor?
